components/LogoBackground.js

```diff
@@ -0,0 +1,23 @@
+const LogoBackground = ({ size = 820, offsetTop = 0, left, right, bottom }) => (
+  <div className='top-bg-container' style={{ height: size + offsetTop }}>
```

Comment:
Since this is a presentational component, let’s add `aria-hidden` in here as well.